package.json

```diff
@@ -2,29 +2,20 @@
   "name": "Yeoman.io",
   "private": true,
   "devDependencies": {
+    "autoprefixer": "^5.0.0",
     "grunt": "^0.4.5",
-    "grunt-autoprefixer": "^3.0.0",
     "grunt-build-control": "^0.4.0",
     "grunt-cli": "^0.1.13",
     "grunt-concurrent": "^1.0.0",
     "grunt-contrib-clean": "^0.6.0",
-    "grunt-contrib-concat": "^0.5.0",
-    "grunt-contrib-connect": "^0.10.1",
-    "grunt-contrib-copy": "^0.8.0",
     "grunt-contrib-csslint": "^0.4.0",
-    "grunt-contrib-cssmin": "^0.12.2",
-    "grunt-contrib-htmlmin": "^0.4.0",
-    "grunt-contrib-imagemin": "^0.9.2",
     "grunt-contrib-jshint": "^0.11.1",
-    "grunt-contrib-sass": "^0.9.2",
-    "grunt-contrib-uglify": "^0.9.1",
-    "grunt-contrib-watch": "^0.6.1",
-    "grunt-filerev": "^2.1.2",
     "grunt-jekyll": "^0.4.2",
-    "grunt-svgmin": "^2.0.0",
-    "grunt-usemin": "^3.0.0",
+    "grunt-livestyle": "^1.5.1",
+    "grunt-reduce": "^2.0.0",
     "jshint-stylish": "^1.0.0",
     "load-grunt-tasks": "^3.1.0",
+    "svgo": "^0.4.5",
```

Comment:
It's an optional peer dependency of grunt-reduce. If it is available SVG's will be optimized with it.